com.google.Chrome.yaml

```diff
@@ -0,0 +1,94 @@
+app-id: com.google.Chrome
+desktop-file-name-prefix: '(Testing) '
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: chrome
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - '--device=dri'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--require-version=1.8.0'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.freedesktop.UPower'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--filesystem=home'
+modules:
+  - shared-modules/libsecret/libsecret.json
+
+  - name: zypak
+    sources:
+      - type: git
+        url: https://github.com/refi64/zypak
+        branch: spawn-strategy
+        commit: 0c58167d2feb1e8065eafa3df06e7f63043a8a15
+      # - type: dir
+      #   path: ../../code/zypak
```

Comment:
I guess you will remove that later?!

Reply:
Yup! It's just easier here for now while I work on them both.